src/main/java/athena/Parser.java

```diff
@@ -173,7 +173,7 @@ private static int getNumber(String taskInfo) throws EditNoIndexException {
     public static Command parseListCommand(String taskInfo, int importancePos, int forecastPos)
             throws InvalidCommandException {
         String importanceDefault = "ALL";
-        String forecastDefault = "TODAY";
+        String forecastDefault = "WEEK";
```

Comment:
Remember to update this new default value in user guide

Reply:
Oh thanks I forgot about that.